### ISSUE TITLE
update genFormula for use with double dot notation

### ISSUE THREAD

**Issue by elinsooon** [kgoldfeld/simstudy#156]
allow for coefficients to be double dot in addition to numeric

**Comment by kgoldfeld** [comment]
Yes - we want something like `genFormula(c(5, ”a”, “b”), c(“x”, “y”))` to generate `“5 + ..a * x + ..b * y”`.

**Comment by elinsooon** [comment]
I believe that I implemented this correctly but I'm having some issues with the testing. All of my tests pass when run through the console, but when run from the test file, I get errors about the external (double dot) variables not being defined, even though they are. Perhaps this is something to do with the environment that tests are run inside that I don't understand, but I think that the functionality of genFormula is all there.
